tests/load/pipeline/test_stage_loading.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+import pytz
+import datetime  # noqa: 251
+
+from pathlib import Path
+import dlt, os
+from dlt.common import json
+
+from tests.load.pipeline.test_merge_disposition import github
+from tests.load.pipeline.utils import  load_table_counts
+from tests.pipeline.utils import  assert_load_info
+from tests.common.utils import all_data_types
+
+staging_combinations_fields = "destination,file_format,bucket"
+staging_combinations = [
+    ("redshift","parquet","s3://dlt-ci-test-bucket"),
```

Comment:
I think tests for filesystem define bucket names (or there is a method to get it from the config). maybe we should reuse? if not then good